test/bin/pyutils/build_bootc_images.py

```diff
@@ -157,9 +159,16 @@ def process_containerfiles(groupdir, dry_run=False):
                 continue
 
         common.print_msg(f"Processing {cf_path}")
-        common.run_command(
-            ["podman", "build", "-t", cf_outname, "-f", cf_path,
-                os.path.join(IMAGEDIR, "rpm-repos")], dry_run)
+        # Run the container build command
+        build_args = [
+            "podman", "build",
+            "--authfile", PULL_SECRET,
```

Comment:
FYI: Alternative would be to copy the pull secret(s) to ~/.docker/config.json, but we already expect ~/.pull-secret.json in other places (e.g. rebase.sh) so I guess it's fine

Reply:
I'd prefer using the pull secret directly, not to mess up the user configuration files.